homeassistant/components/netatmo/camera.py

```diff
@@ -219,6 +223,7 @@ def device_state_attributes(self):
             "is_local": self._is_local,
             "vpn_url": self._vpnurl,
             "local_url": self._localurl,
+            "light_state": self._light_state,
```

Comment:
Why add this as an attribute and not as an entity on its own?

Reply:
The camera light is already exposed as a light platform. But since that can only have on/off as state and the light belongs to the camera anyway it was the simplest to do. I'm am open to other suggestions if you think it doesn't fit.

Reply:
We spoke on discord, it has a tri-state (possibility). On/Off/Auto, which isn't fully covered by the light, as Home Assistant doesn't have such a state.